### DISCUSSION TITLE
Migration from  sqllite3  to mysql

### DISCUSSION THREAD

**Discussion by Kenny-SZ** [grafana/grafana#66633] [Alerting]
When I added some alert rules into  grafana， it said: "database is locked". So I exported data from sqllite3, and imported into mysql.

Everything is OK except for alert rule. It shows: “scheduler failed to update alert rules" err="failed to get alert rules: invalid character 'c' after object key:value pair"

Even when I create an alert rule, it shows: "Failed to save rule: failed to get group alert rules: invalid character 'c' after object key:value pair"

grafana：v9.1.0
mysql：5.7.29

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I feel like this is a question best served by asking our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request a feature.
